iampolicymanagementv1/iam_policy_management_v1_examples_test.go

```diff
@@ -0,0 +1,399 @@
+// +build examples
+
+/**
+ * (C) Copyright IBM Corp. 2021.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package iampolicymanagementv1_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/IBM/platform-services-go-sdk/iampolicymanagementv1"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"math/rand"
+	"os"
+	"strconv"
+)
+
+// Below are examples on how to use IAM Policy Management service
+//
+// The following environment variables are assumed to be defined when running examples below:
+//
+// IAM_POLICY_MANAGEMENT_URL=https://iam.cloud.ibm.com
+// IAM_POLICY_MANAGEMENT_AUTH_TYPE=iam
+// IAM_POLICY_MANAGEMENT_AUTH_URL=https://iam.cloud.ibm.com/identity/token
+// IAM_POLICY_MANAGEMENT_APIKEY= <YOUR_APIKEY>
+// IAM_POLICY_MANAGEMENT_TEST_ACCOUNT_ID= <YOUR_ACCOUNT_ID>
+//
+// Alternatively, above environment variables can be placed in a "credentials" file
+
+const externalConfigFile = "../iam_policy_management.env"
+
+var (
+	// TODO: Align
+	iamPolicyManagementService *iampolicymanagementv1.IamPolicyManagementV1
+	config       map[string]string
+	configLoaded bool = false
+
+	testUserId        = "IBMid-GoSDK" + strconv.Itoa(rand.Intn(100000))
+	testServiceName   = "iam-groups"
+	testAccountId      string
```

Comment:
Consider changing variable names that show up in example code snippets to use `example` instead of `test`.